Creates the blog post for releasing Data Prepper 2.7.0 (#2654)
opensearch-project/project-website: _posts/2024-03-26-Announcing-Data-Prepper-2.7.0.md

For more information, please see the [`aws` extension](https://opensearch.org/docs/latest/data-prepper/managing-data-prepper/configuring-data-prepper/#aws-extension-plugins) documentation.

> This feature is experimental while we work to add more support for refreshing certain fields and dynamically updating those fields.

Reviewer: ```suggestion
Note that this feature is currently experimental, and we are working to add support for refreshing and dynamically updating certain fields.
```

> In particular, the `opensearch` sink and the `kafka` plugins do not automatically refresh secrets.

Reviewer: I'm assuming "plugins" is intentionally plural.

Author: Yes, there are two `kafka` plugins.

## Other features

* Data Prepper can now parse XML data in fields using the `parse_xml` processor.
* The new `parse_ion` processor can parse fields in the [Amazon Ion format](https://amazon-ion.github.io/ion-docs/).
* Some users have fields that are gzip-compressed at the field level. These users can decompress those fields using the `decompress` processor.
* Data Prepper can now join strings from multiple strings, including with a delimiter.
* The new `select_entries` processor allows users to select only the necessary fields from events. This can simplify how users filter unnecessary data.

> * Users who wish to reduce the size of fields in OpenSearch can use the `truncate` processor which truncates strings to a configurable maximum length.

Reviewer: ```suggestion
* Users who wish to reduce the size of fields in OpenSearch can use the `truncate` processor, which truncates strings to a configurable maximum length.
```
